shared-bindings/microcontroller/__init__.c

```diff
@@ -171,6 +177,7 @@ STATIC const mp_rom_map_elem_t mcu_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_enable_interrupts), MP_ROM_PTR(&mcu_enable_interrupts_obj) },
     { MP_ROM_QSTR(MP_QSTR_on_next_reset), MP_ROM_PTR(&mcu_on_next_reset_obj) },
     { MP_ROM_QSTR(MP_QSTR_reset), MP_ROM_PTR(&mcu_reset_obj) },
+    { MP_ROM_QSTR(MP_QSTR_sleep), MP_ROM_PTR(&mcu_sleep_obj) },
```

Comment:
Let's not expose this. Instead we should sleep after the python code exits. (Controlled in main.c)

Reply:
I left it there just for testing purpose, will remove it once entering sleep on `code.py` exit is implemented.